### PR TITLE
qt: 24.0rc2 translations update

### PR DESCRIPTION
**Pull request by hebasto**
This PR pulls the recent translations from the [Transifex.com](https://www.transifex.com/bitcoin/bitcoin) using the [`bitcoin-core/bitcoin-maintainer-tools/update-translations.py`](https://github.com/bitcoin-core/bitcoin-maintainer-tools/blob/main/update-translations.py) tool, and it is supposed to be merged just before `v24.0rc2` tagging.

### CODE REVIEW

**Comment by hebasto** [comment]
Going to keep this PR continuously updated.

cc @fanquake @jarolrod @luke-jr @wtogami

File: src/qt/locale/bitcoin_fr.ts [hunk 1]
```diff
@@ -3941,26 +3967,17 @@ Note : Les frais étant calculés par octet, un taux de frais de « 100 satoshi
...
-        <translation type="unfinished">pas dans la réserve de mémoire</translation>
-    </message>
     <message>
         <source>abandoned</source>
+        <extracomment>Text explaining the current status of a transaction, shown in the status field of the details window for this transaction. This status represents an abandoned transaction.</extracomment>
         <translation type="unfinished">abandonnée</translation>
     </message>
     <message>
         <source>%1/unconfirmed</source>
+        <extracomment>Text explaining the current status of a transaction, shown in the status field of the details window for this transaction. This status represents a transaction confirmed in at least one block, but less than 6 blocks.</extracomment>
```
Comment (darosior) [inline src/qt/locale/bitcoin_fr.ts:3980]: Unconfirmed is for not confirmed deeply enough transactions?

File: src/qt/locale/bitcoin_fr.ts [hunk 2]
```diff
@@ -3663,15 +3702,18 @@ Note : Les frais étant calculés par octet, un taux de frais de « 100 satoshi
...
     </message>
-    <message>
-        <source>Payment request expired.</source>
-        <translation type="unfinished">La demande de paiement a expiré.</translation>
+    <message numerus="yes">
+        <source>Estimated to begin confirmation within %n block(s).</source>
+        <translation type="unfinished">
+            <numerusform />
+            <numerusform />
+        </translation>
```
Comment (darosior) [inline src/qt/locale/bitcoin_fr.ts:3716]: Doesn't look like it replaces the previous traduction?

**Review by darosior — COMMENTED** [review]
Went through the french diff, mostly good and nothing insane. some things, like errors, are probably not worth translating: it's awkward and not understandable to user anyways, it's just supposed to be pasted in a bug report i guess? (And if translated it will be harder to debug for those treating the bug report)

**Comment by hebasto** [comment]
@darosior 

> Went through the french diff, mostly good and nothing insane. some things, like errors, are probably not worth translating: it's awkward and not understandable to user anyways, it's just supposed to be pasted in a bug report i guess? (And if translated it will be harder to debug for those treating the bug report)

This PR just pulls translations from Transifex website using a script (no manual interaction). To amend or discuss particular translated strings, it would better to use Transifex directly.

**Comment by hebasto** [comment]
Pulled the recent translations updates from the Transifex website.

**Comment by hebasto** [comment]
Pulled the recent translations updates from the Transifex website.

**Comment by hebasto** [comment]
Updated 69334a00a6b986c9b6be6257bcbd85df0c752b89 -> efa4e3b71fa23d90974dca7c8b7b01eaa1600be2 ([pr26141.02](https://github.com/hebasto/bitcoin/commits/pr26141.02) -> [pr26141.03](https://github.com/hebasto/bitcoin/commits/pr26141.03), [diff](https://github.com/hebasto/bitcoin/compare/pr26141.02..pr26141.03)):

- pulled the recent translations updates from the Transifex website

**Comment by hebasto** [comment]
Updated efa4e3b71fa23d90974dca7c8b7b01eaa1600be2 -> a0d5e395319f0cb1a299e6c7579ec7e886b30e17 ([pr26141.03](https://github.com/hebasto/bitcoin/commits/pr26141.03) -> [pr26141.04](https://github.com/hebasto/bitcoin/commits/pr26141.04), [diff](https://github.com/hebasto/bitcoin/compare/pr26141.03..pr26141.04)):

- pulled the recent translations updates from the Transifex website

**Comment by hebasto** [comment]
Updated a0d5e395319f0cb1a299e6c7579ec7e886b30e17 -> d6dc3cb9d63a3595c3a76c0c8884c712ef6237ca ([pr26141.04](https://github.com/hebasto/bitcoin/commits/pr26141.04) -> [pr26141.05](https://github.com/hebasto/bitcoin/commits/pr26141.05), [diff](https://github.com/hebasto/bitcoin/compare/pr26141.04..pr26141.05)):

- pulled the recent translations updates from the Transifex website

**Comment by hebasto** [comment]
@fanquake 
> If there are any more changes can you pull them in here. Planning on merging this shortly.

Done. Thank you.

**Pull request merged**